services/worker-manager/src/providers/aws.js

```diff
@@ -1,17 +1,32 @@
 const {Provider} = require('./provider');
 const aws = require('aws-sdk');
 const _ = require('lodash');
+const taskcluster = require('taskcluster-client');
 
 class AwsProvider extends Provider {
-  constructor({WorkerPool, instancePermissions, apiVersion, region}) {
-    super({WorkerPool});
+  constructor({
+    WorkerPool,
+    instancePermissions,
+    apiVersion,
+    region,
+    estimator,
+    notify,
+    project,
```

Comment:
I think `project` is a google-specific thing iiuc how this is being used. Probably here it would be `account`?

Reply:
oh, right!

Reply:
I have a bug, or a note somewhere, to change the constructor syntax so that it doesn't mix provider config items (of which project is one) with parameters passed to all providers (estimator, notify, etc.)